src/components/Questions/Header.vue

```diff
@@ -32,16 +32,16 @@ export default defineComponent({
       } as IconButtonTitleConfig,
     });
 
-    function skipItem() {
-      context.emit("skip-item");
+    function skipQuestion() {
```

Comment:
Now why is this in this PR? :(

Reply:
I realised that we are not keeping it generic as content before and so, just changed to Question so that Saloni and others don't get confused. As we are working on a PR, if we are finding minor things that anyway need to be corrected, can't we just do it in the same PR - given they are super small?

Reply:
I'm very much of the opinion that No. Because imagine if it was a big PR with some small change that you were trying to put in. Now this small change will be held up by the big PR. If you just make another PR with only the small change, it will get merged immediately and everyone else will get access to the updated code sooner!

Reply:
Can resolve this conversation, but I am very much of the opinion that super small changes especially need their own PR because of reason above